test/house_test.rb

```diff
@@ -0,0 +1,129 @@
+require 'minitest/autorun'
+require 'minitest/pride'
+require './lib/room'
+require './lib/house'
+
+class HouseTest < Minitest::Test
+  def test_it_exists
```

Comment:
I would consider using a `setup` method with instance variables to create your objects to save you some time and avoid duplicating work!